Pear/View/FeedbackView.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  FeedbackView.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/21/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+import UIKit
+
+class FeedbackView: UIView {
+
+    // MARK: - Private View Vars
+    private var arrowBackgroundView = UIView()
+    private var arrowView = UIView()
+    private let feedbackTableView = UITableView()
+    private let feedbackBackgroundView = UIView()
+
+    // MARK: - Private Data Vars
+    private let feedbackOptions = ["Send feedback", "Contact us", "Report user"]
+    private let size = 20
+
+    init() {
+        super.init(frame: .zero)
+        setUpViews()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setUpViews() {
+        arrowBackgroundView.backgroundColor = .clear
+        addSubview(arrowBackgroundView)
+
+        let path = CGMutablePath()
+        path.move(to: CGPoint(x: 0, y: size))
+        path.addLine(to: CGPoint(x:size/2, y: size/2))
+        path.addLine(to: CGPoint(x:size, y:size))
+        path.addLine(to: CGPoint(x:0, y:size))
+        let shape = CAShapeLayer()
+        shape.path = path
+        shape.fillColor = UIColor.white.cgColor
+        arrowView.layer.insertSublayer(shape, at: 0)
+        arrowBackgroundView.addSubview(arrowView)
+
+        feedbackBackgroundView.backgroundColor = .white
+        addSubview(feedbackBackgroundView)
+
+        feedbackTableView.backgroundColor = .backgroundWhite
+        feedbackTableView.separatorStyle = .none
+        feedbackTableView.showsVerticalScrollIndicator = false
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.isScrollEnabled = false
+        feedbackTableView.register(FeedbackMenuTableViewCell.self, forCellReuseIdentifier: FeedbackMenuTableViewCell.reuseIdentifier)
```

Comment:
if we remove FeedbackCell.swift and directly render cell in this file we wont have to do this

Reply:
Agreed. For only 3 rows there's no point in a table. Just three constrained buttons would be much simpler.